libs/@guardian/source-react-components/vendor/icons/SvgEdit.tsx

```diff
@@ -17,7 +17,7 @@ const Svg = ({ size }: IconProps): EmotionJSX.Element => (
 		<path
 			fillRule="evenodd"
 			clipRule="evenodd"
-			d="m19.43 6.475-.325-.3-5.65 5.675-.35-.375L21.605 3l1.4 1.4-8.475 8.5-.775-.75 5.675-5.675ZM12.255 14.1l-.375-.35.85-1.9 1.425 1.4-1.9.85ZM1.975 21 1 19.975V8l.975-1H11v1l-1 1H3v10h16v-5l.975-1H21v6.975L19.975 21h-18Z"
+			d="M18.4045 1L4.39711 15.1991L6.51843 17.349L8.63975 19.4989L20.7587 7.21452L16.5187 2.91747L17.2485 2.17788L21.4883 6.47487L22.6758 5.27123L18.4045 1ZM14.7821 6.71093L5.40266 16.2168L6.07255 16.8957L15.452 7.38985L14.7821 6.71093ZM3.68889 15.9144L2.63044 16.9916L2 21.3691L2.55378 21.9303L6.87308 21.2914L7.93153 20.2142L3.68889 15.9144Z"
```

Comment:
Could this be cleaned up a bit with SVGO? It seems to have quite a lot of precision after the decimal point